source.lua

```diff
@@ -1,10 +1,28 @@
 --@ module = true
 local repeatUtil = require('repeat-util')
+local json = require('json')
+local persist = require('persist-table')
 
-liquidSources = liquidSources or {}
+local GLOBAL_KEY = 'source' -- used for state change hooks and persistence
+
+g_state = g_state or {}
 
 local sourceId = 'liquidSources'
 
+enabled = enabled or false
+
+function isEnabled()
+    return enabled
```

Comment:
now that I look at it, you might be better off following a model closer to `prioritize`. this tool will be "enabled" if there are sources in the state map.

Reply:
that looks like it was the golden ticket